apiserver/facades/agent/deployer/deployer_test.go

```diff
@@ -60,7 +61,7 @@ type deployerSuite struct {
 	subordinate0 *state.Unit
 
 	resources *common.Resources
-	deployer  *deployer.DeployerAPI
+	coreunit  *deployer.DeployerAPI
```

Comment:
**question:** Why this change ? How it make it more readable ? (out of context, deployer in deployer suite make sense. coreunit doesn't, and may clash with a package, like core/unit)

Reply:
Oops! My IDE renamed the wrong variable here it seems!

Reverting